cxx/transition_true_value.hh

```diff
@@ -0,0 +1,105 @@
+// Copyright 2024
+// See LICENSE.txt
+
+#pragma once
+
+#include <algorithm>
+#include <random>
+#include <string>
+#include <unordered_map>
+
+#include "emissions/base.hh"
+#include "noisy_relation.hh"
+#include "relation.hh"
+
+typedef int T_item;
+typedef std::vector<T_item> T_items;
+
+// Updates the estimate of a latent value conditioned on noisy observations.
+template <typename ValueType>
+void transition_true_value(
```

Comment:
Slight preference for "transition_latent_value" over "transition_true_value".

Reply:
I like that better too, done.